src/_P128_NeoPixelBusFX.ino

```diff
@@ -174,13 +175,14 @@ boolean Plugin_128(uint8_t function, struct EventStruct *event, String& string)
       break;
     }
 
-    # ifdef ESP32
     case PLUGIN_SET_DEFAULTS:
     {
-      PIN(0) = -1; // None
+      # ifdef ESP32
+      PIN(0) = -1;                  // None
```

Comment:
Why only set it for ESP32?

Reply:
It is not configurable for ESP8266, fixed on GPIO 2. (for the version of the library I've selected/set fixed, later versions don't properly compile anymore)